go/ct/driver/cli/flags.go

```diff
@@ -62,12 +63,16 @@ var SeedFlag = &seedFlagType{
 	cli.Uint64Flag{
 		Name:    "seed",
 		Aliases: []string{"s"},
-		Usage:   "seed for the random number generator",
+		Usage:   "seed for the random number generator. If not provided, the seed is the current time stamp",
```

Comment:
now that I think about it, the same could have been done with a default value:https://cli.urfave.org/v2/examples/flags/

```go
cli.Uint64Flag{
		Name:    "seed",
		Aliases: []string{"s"},
		Value:    uint64(time.Now().UnixNano()),
		Usage:   "seed for the random number generator. If not provided, the seed is the current time stamp",
```